tests/post_training/data/wc_reference_data.yaml

```diff
@@ -7,35 +7,36 @@ tinyllama_data_aware_backend_OV:
   num_int4: 94
   num_int8: 124
 tinyllama_data_aware_awq_stateful_backend_OV:
-  metric_value: 0.85616
+  metric_value: 0.85571
   num_int4: 94
   num_int8: 124
 tinyllama_data_aware_awq_scale_estimation_backend_OV:
-  metric_value: 0.85502
+  metric_value: 0.86355
   num_int4: 94
   num_int8: 124
 tinyllama_data_aware_awq_scale_estimation_stateful_backend_OV:
-  metric_value: 0.85502
+  metric_value: 0.86355
   num_int4: 94
   num_int8: 124
 tinyllama_int8_data_free_backend_TORCH:
   metric_value: 0.95624
   num_int4: 0
   num_int8: 312
 tinyllama_data_aware_gptq_backend_OV:
-  metric_value: 0.87134
+  metric_value: 0.86521
   num_int4: 94
   num_int8: 124
   metrics_xfail_reason: "Issue-148819"
 tinyllama_scale_estimation_per_channel_backend_OV:
-  metric_value: 0.80873
+  metric_value: 0.80853
```

Comment:
Shouldn't it be marked by xfail, since 197 build failed with 0.80873 reference?

Reply:
In agreement with @l-bat, I updated the metrics. @l-bat, please, confirm that the result is reproducible on different hardware.

Reply:
This test has become unstable on openvino-nightly again (details at ticket: 152627)

Reply:
I do not update openvino-nightly metrics.